Gemfile

```diff
@@ -54,6 +60,7 @@ group :test do
   gem 'selenium-webdriver'
   # Easy installation and use of web drivers to run system tests with browsers
   gem 'webdrivers'
+  gem 'database_cleaner-active_record'
```

Comment:
I have not used database cleaner before. Please tell me why is this gem being used?

Reply:
This is to clean the test data after every spec run. This is not needed starting from Rails 5 onwards as Rails supports cleaning of database fixtures after every spec run.

Reply:
Just shows how ancient my knowledge is.